Documentation/Settings.cfg

```diff
@@ -23,6 +23,10 @@ project_discussions  =
 
 use_opensearch       =
 
+[sphinx_object_types_to_add]
+
+cobj-text =  cobj-text // cobj-text // Content object TEXT
```

Comment:
Can you add a comment what this syntax means?

Reply:
See https://github.com/TYPO3-Documentation/TYPO3CMS-Guide-HowToDocument/pull/310